accepted/2020/reproducible-builds.md

```diff
@@ -0,0 +1,238 @@
+
+However, one largely unsolved problem is the link between source code and binaries. While a significant portion of packages on nuget.org are open source and often link to a commit in a GitHub repo, there is no way to ensure that the source code at that commit actually matches the binaries submitted to nuget.org. In other words, just because a developer can trust the source code doesn’t mean they can trust the binaries came from that source. And while nuget.org scans packages for virus and malware, there is no way to detect all possible malware. For all they know, someone, or something, [injected a bitcoin miner](https://www.theregister.com/2018/11/26/npm_repo_bitcoin_stealer/).
+
+Also, should package consumers find a zero-day exploit, they usually have no way to patch the binaries themselves. They must find the corresponding repo, get the necessary dependencies for building it, somehow find the correct version of the code, patch it, and then rebuild. Not an easy task under ideal circumstances, much less viable when time is of the essence.
+
+To address these problems, we will create a set of tooling that for each managed binary file in a NuGet package, will be able to enumerate all of the input sources used to create it, validate they are accessible and validate that the resulting binary matches was produced using those sources files. In doing so, we don't need to trust the original build environment, as we're able to independently create the managed file from the input sources.
+
+## Scenarios and User Experience
+
+### Customer Cohorts
+
+- Package consumers who wish to manually check results on their own.
+- Package authors who wish to verify their package is reproducible.
+- Package hosts, such as NuGet.org and Terrapin, who wish to validate packages on upload and present the status as part of overall package/project health.
+
+### End-to-end scenarios
+
+#### Developer or InfoSec that wants to validate on their own
+
+An internal package security team is wants to reduce their supply chain risk to their apps and services. They want to make sure that the binaries being used can be traced to sources so they can run security scans on the source code. The packages being validated are produced from a mix of public and internal sources. The team sets up infrastructure to run the validation tool for each package they use.
+
+Due to organization security requirements, based on likely forthcoming NIST standards, they must be able to trace the source and rebuild all packages being used.
+
+1. For each managed binary in the NuGet package, call a tool to check the status. Result is one of: not buildable, buildable but not verifiable (builds but not the same deterministic output), or deterministically reproducible.
```

Comment:
"buildable but not verifiable" : Is there a reason for this specific wording? It seems to indicate that tool could build but may have failed in some way to do a verification step. Maybe "buildable but not verified". Could even go for more severe wording like "built but failed verification"

Reply:
In general the verb we were using for the provenance step was "verify" hence "verifiable" makes sense in that context. 

Personally I'm not wedded to "verify" as the verb here but we also couldn't find another one that read well.